src/Task.ts

```diff
@@ -313,19 +336,40 @@ export function loadOrCreateAcceptanceList(tokensAcceptanceListId: string): Acce
 
   if (acceptanceList === null) {
     acceptanceList = new AcceptanceList(tokensAcceptanceListId)
-    acceptanceList.task = tokensAcceptanceListId
+    acceptanceList.taskConfig = tokensAcceptanceListId
     acceptanceList.type = 'DenyList'
     acceptanceList.tokens = []
   }
 
   return acceptanceList
 }
 
+export function loadOrCreateTaskConfig(taskId: string): TaskConfig {
+  let taskConfig = TaskConfig.load(taskId)
+
+  if (taskConfig === null) {
+    taskConfig = new TaskConfig(taskId)
+    taskConfig.acceptanceList = loadOrCreateAcceptanceList(taskId).id
+    taskConfig.previousBalanceConnector = '0x0000000000000000000000000000000000000000000000000000000000000000'
+    taskConfig.nextBalanceConnector = '0x0000000000000000000000000000000000000000000000000000000000000000'
+    taskConfig.gasPriceLimit = BigInt.zero()
+    taskConfig.priorityFeeLimit = BigInt.zero()
+    taskConfig.timeLockDelay = BigInt.zero()
+    taskConfig.timeLockExecutionPeriod = BigInt.zero()
+    taskConfig.timeLockExpiration = BigInt.zero()
+    taskConfig.txCostLimitPct = BigInt.zero()
+    taskConfig.txCostLimit = BigInt.zero()
+    taskConfig.save()
+  }
+
+  return taskConfig as TaskConfig
+}
+
 export function parseAcceptanceType(op: i32): string {
   if (op == 0) return 'DenyList'
   else return 'AllowList'
 }
 
-export function getTaskCustomConfigId(task: Task, token: Address): string {
+export function getTaskCustomConfigId(task: TaskConfig, token: Address): string {
   return task.id.toString() + '/' + token.toHexString()
```

Comment:
Let's use proper names:

```suggestion
export function getTaskCustomConfigId(taskConfig: TaskConfig, token: Address): string {
  return taskConfig.id.toString() + '/' + token.toHexString()
```